contracts/ModelRepository.sol

```diff
@@ -73,14 +73,22 @@ contract ModelRepository {
     models.push(newModel);
   }
 
+  function incentiveCalculate(uint bounty, uint totalError, uint solvedError) returns(uint total) {
+    total = ((bounty/totalError) * solvedError);
+    return total;
```

Comment:
could do these two lines in one and remove the outer braces

Reply:
Yes. Updating :+1: